src/components/route-anatomy/styles.module.css

```
@@ -0,0 +1,70 @@
+}
+
+[data-theme='dark'] .anatomy {
+    background: #282a36;
+    color: #f8f8f2;
+    border-color: var(--ifm-color-emphasis-300);
+}
+
+.anatomy a {
+    font-weight: 600;
+    text-decoration: none;
+    border-bottom: 2px dashed currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)